packages/api/tests/endpoints/test-rules.js

```diff
@@ -525,6 +525,248 @@ test('POST creates a rule in all data stores', async (t) => {
   t.like(esRecord, translatedPgRecord);
 });
 
```

Comment:
Add SQS tests related to the ticket back.
Also add SNS/Kinesis tests which got dropped in release 16 back.